packages/stores/src/account/osmosis/index.ts

```diff
@@ -2339,6 +2339,59 @@ export class OsmosisAccountImpl {
     );
   }
 
+  /**
+   * Method to withdraw delegation rewards and delegate to validator set - staking collect and reinvest
+   * @param coin The coin object with denom and amount to delegate.
+   * @param memo Transaction memo.
+   * @param onFulfill Callback to handle tx fulfillment given raw response.
+   */
+  async sendWithdrawDelegationRewardsAndSendDelegateToValidatorMsg(
```

Comment:
Tiny detail; it's nice to end the call with `Msgs` instead of `Msg` to signal the fact that it's a multi message. Even though here, it may be obvious with the `...And...`

Also, can we add `Set` after `Validator`